website/core/Footer.js

```diff
@@ -1,14 +1,16 @@
 const React = require("react");
 
+const DEFAULT_LANGUAGE = "en";
+
 class Footer extends React.Component {
-  docUrl(doc, language = "en") {
+  docUrl(doc, language) {
     const baseUrl = this.props.config.baseUrl;
-    return baseUrl + "docs/" + (language ? language + "/" : "") + doc;
+    return baseUrl + "docs/" + (language ? language : DEFAULT_LANGUAGE) + "/" + doc;
```

Comment:
Thoughts on centralizing these helpers (and default language constant):
https://github.com/babel/website/blob/master/website/pages/en/index.js#L5